packages/razzle-dev-utils/prettyNodeErrors.js

```diff
@@ -0,0 +1,35 @@
+const {
+  formatExecError,
+  separateMessageFromStack,
+} = require('jest-message-util');
+
+function pretty(error) {
+  return `\n${formatExecError(
+    error,
+    { rootDir: process.cwd() },
+    {},
+    undefined,
+    true
+  )}`;
+}
+
+function usePrettyErrors(transform) {
+  const { prepareStackTrace } = Error;
+
+  Error.prepareStackTrace = (error, trace) => {
+    const prepared = prepareStackTrace
+      ? separateMessageFromStack(prepareStackTrace(error, trace))
+      : error;
+    const transformed = transform ? transform(prepared) : prepared;
+
+    return pretty(transformed);
+  };
+}
+
+// Currently needed to fix sourcemap path
+const stackTransform = ({ stack = '', ...rest }) => ({
+  stack: stack.replace('/build/webpack:', ''),
```

Comment:
I wonder if this has to do with some sort of memory filesystem within webpack?

Reply:
Ahh it looks like it does. It's called "sourcemap namespacing." I'm not 100% sure we should remove this. 

https://github.com/webpack/webpack/blob/fe0eb45079019281cb536ef887adf301222e5a63/test/configCases/source-map/namespace-source-path/index.js

@nathanforce's suggestion in Slack about copying CRA is a good idea i think. Checkout `formatWebpackErrors.js` in `react-dev-utils`

Reply:
Here's where CRA pulls it out i think. https://github.com/facebook/create-react-app/blob/next/packages/react-dev-utils/formatWebpackMessages.js#L112

Reply:
Let's add a comment in the source that references the line of CRA with

```js
// Clean up Webpack's sourcemap namespacing in error stacks
// @see https://github.com/facebook/create-react-app/blob/next/packages/react-dev-utils/formatWebpackMessages.js#L112
```